readme.md

```diff
@@ -330,157 +451,219 @@ let prepareTo = "die"
+```
+
+If the function is multi-line, you need to explicitly return. Otherwise, you get this...
+
+```js
+const divide = (dividend, divisor) => {
+  dividend / divisor
+}
+
+divide(2,3)
+// => undefined
+```
+
+Though the single line return can be faked by wrapping the expression in parentheses...
+
+```js
+const divide = (dividend, divisor) => (
+  dividend / divisor
+)
+```
+
+#### Context in Arrow Functions
+
+Arrow functions also have the benefit of maintaining the context (`this`) of where the function is defined...
+
+Context works differently in arrow functions than it does in ES5. Consider the below example.
+
+<details>
+  <summary><strong>What is the result of running <code>this.temperature++</code>? Why?</strong></summary>
+
+  > `setInterval`'s callback is what we call an "unbound" function (i.e., it is stand-alone and not contained in an object). That means that it will default to a global context, which in the browser is the `window` object.
+  >
+  > When we call `this.temperature++`, Javascript looks for a `temperature` property on the `window` object. This does not occur naturally.
+  >
+  > If we `console.log(window.temperature)`, we see that it is `NaN`. This is the result of incrementing an `undefined` value by 1.
+
+</details>
+
+<br>
 
 ```js
 var pizza = {
   temperature: 0,
   toppings: ["cheese", "ham", "pineapple"],
   bake() {
     setInterval(function() {
-      this.temperature++ // doesnt work because this is GLOBAL. The setInterval function belongs to the window object.
+      this.temperature++
     }, 1000)
   }
 }
 
-// vs ES6
+pizza.bake()
+console.log(pizza.temperature)
+```
+
+It would be nice, however, if we could use `this` in a similar way and have it work.
+
+Let's try the same thing, except this time we will replace `setInterval`'s callback function with an arrow function.
+
+<details>
+  <summary><strong>What is the result of running <code>this.temperature++</code>? What might this tell us about context when using arrow functions?</strong></summary>
+
+  > It incremented the `temperature` value stored in pizza!
+  >
+  > Perhaps unbound arrow functions do not default to the global scope...
+
+</details>
+
+<br>
 
+```js
 var pizza = {
   temperature: 0,
   toppings: ["cheese", "ham", "pineapple"],
   bake() {
-    setInterval( () => {
+    setInterval(() => {
       this.temperature++
     }, 1000)
   }
 }
 
 pizza.bake();
-pizza.temperature //will display the return value of setInterval, which is the ID value of the timer that was set
+console.log(pizza.temperature)
 ```
 
-Additionally, the `return` statement is not needed with single line arrow functions. There is an implicit return.
+An arrow function's context is defined by its enclosing context (i.e., where it is defined). In this case, that enclosing context is the `pizza` object. An arrow function does not create its own context, unlike an unbound `function` which would have created its own context -- `window`.
 
```

Comment:
You could add that it doesn't introduce a new context but introduces a new scope